pkg/kubelet/lifecycle/handlers.go

```diff
@@ -55,6 +56,9 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
 		var buffer bytes.Buffer
 		output := ioutils.WriteCloserWrapper(&buffer)
 		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false)
+		if err != nil {
+			glog.V(1).Infof("Liveness-Probe for Container %q in Pod %q failed - %q", container.Name, pod.Name, buffer.String())
```

Comment:
This is for "lifecycle" hooks -- PreStop and PostStart container hooks. It has nothing to do with liveness probes, which already logs output on error.